sonar-scanner-cli.yaml

```
@@ -0,0 +1,52 @@
+package:
+  name: sonar-scanner-cli
+  version: 5.0.1.3006
+  epoch: 0
+  description: Scanner CLI for SonarQube and SonarCloud
+  copyright:
+    - license: LGPL-3.0-or-later
+  dependencies:
+    runtime:
+      - bash
+
+environment:
+  contents:
+    packages:
+      - maven
+      - openjdk-17
+      - openjdk-17-default-jvm
+      - wolfi-base
+      - wolfi-baselayout
+  environment:
+    LANG: en_US.UTF-8
+    JAVA_HOME: /usr/lib/jvm/java-17-openjdk
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/SonarSource/sonar-scanner-cli
+      tag: ${{package.version}}
+      expected-commit: a045fcd1fdadf09d3e533f83ad749a5898e68470
+
+  - uses: patch
+    with:
+      # Bumps okio to mitigate a CVE
+      patches: bumpdeps.patch
+
+  - name: build
+    runs: |
+      mvn clean package
+      mkdir -p ${{targets.destdir}}/usr/bin/sonarqube
+      cp target/sonar-scanner-cli-*.jar ${{targets.destdir}}/usr/bin/sonarqube/sonarscanner-cli.jar
+      chmod +x sonar-scanner-cli && cp sonar-scanner-cli ${{targets.destdir}}/usr/bin/
+
+update:
+  enabled: true
+  github:
+    identifier: sonarqube/sonar-scanner-cli
+    use-tag: true
+
+test:
```

you can check the [maven](https://github.com/wolfi-dev/os/blob/main/maven.yaml#L65-L72) test example 
```suggestion
test:
  environment:
    contents:
      packages:
        - openjdk-17-default-jvm
        - openjdk-17
```

---

This is from a test I ran locally 
```
export PATH='/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin'

[ -d '/home/build' ] || mkdir -p '/home/build'
cd '/home/build'
sonar-scanner-cli --version

exit 0
2024/03/21 06:05:52 INFO INFO: Scanner configuration file: NONE
2024/03/21 06:05:52 INFO INFO: Project root configuration file: NONE
2024/03/21 06:05:52 INFO INFO: SonarScanner 5.0.1-SNAPSHOT
2024/03/21 06:05:52 INFO INFO: Java 17.0.10-internal wolfi (64-bit)
2024/03/21 06:05:52 INFO INFO: Linux 5.15.120+ amd64

```

---

thanks, LGTM